version/version.go

```diff
@@ -1,5 +1,29 @@
 package version
 
+import (
+	"fmt"
+	"runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	sdkVersion "github.com/operator-framework/operator-sdk/version"
+)
+
+var log = logf.Log.WithName("version")
+
+//var needs to be used instead of const for ldflags
 var (
-	Version = "0.0.1"
+	Version            = "0.0.1+git"
+	GoVersion          = fmt.Sprintf("%s %s/%s", runtime.Version(), runtime.GOOS, runtime.GOARCH)
+	OperatorSDKVersion = sdkVersion.Version
+	GitCommit          = "unknown"
```

Comment:
What is the point of this if this is never getting used?

Reply:
Just a nice to have. We can go with either `Version+GitHash` or `GitCommit`. If we are going by `Version+GitHash`, we dont need the entire `GitCommit` hash

Reply:
Not sure why it is a nice to have when it is always set to `unknown`.

Reply:
I remember asking this question in the design meeting and @PratikMahajan had mentioned it will be updated for a release, not sure though how that works here.